sklearn/utils/tests/test_sparsefuncs.py

```diff
@@ -94,7 +94,9 @@ def test_incr_mean_variance_axis():
         rng = np.random.RandomState(0)
         n_features = 50
         n_samples = 10
-        data_chunks = [rng.random_integers(0, 1, size=n_features)
+        #data_chunks = [rng.random_integers(0, 1, size=n_features)
+        #               for i in range(n_samples)]
+        data_chunks = [rng.rand(n_features)
```

Comment:
Well, you need to use a new fused type then, (numeric maybe?)

and then cast it to double `if numeric is int or numeric is long` as before :/